website/project/model.py

```diff
@@ -1655,6 +1685,7 @@ def use_as_template(self, auth, changes=None, top_level=True):
         )
 
         new.save(suppress_log=True)
+        new.add_contributor(contributor=auth.user, permissions=CREATOR_PERMISSIONS, log=False, save=False)
```

Comment:
`save=False` seems to be okay here, even though it's now after the `new.save` call, because the `new` is still getting saved, which is what mattered.